db/migrate/20161210235939_add_order_cycle_id_to_proxy_orders.rb

```diff
@@ -0,0 +1,20 @@
+class AddOrderCycleIdToProxyOrders < ActiveRecord::Migration
+  def up
+    add_column :proxy_orders, :order_cycle_id, :integer
+
+    ProxyOrder.all.each do |proxy_order|
```

Comment:
please, use [find_each](https://apidock.com/rails/v4.2.7/ActiveRecord/Batches/find_each) instead of `all`. It can have pretty bad consequences.

Reply:
👍